Documentation/Stage-1-Introduction/Index.rst

```diff
@@ -83,9 +83,11 @@ Theoretical goals
 
 By the end of this stage, you should know the following:
 
-* Item 1
-* Item 2
-* Item 3
+* Install and configure DDEV on your local computer
+* Setup a Git repository for your project
+* Use Composer to install a TYPO3 project in a DDEV container
+* Configure a TYPO3 project
+* Install and configure Bootstrap Package extension for newly the created TYPO3 project
```

Comment:
```suggestion
* How to install and configure DDEV on your local computer
* How to setup a Git repository for your project
* How to use Composer to install a TYPO3 project in a DDEV container
* How to configure a TYPO3 project
* How to install and configure the Bootstrap Package extension for a TYPO3 project
```

On "how to configure a TYPO3 project", maybe add a few words that limits the scope. For example "to run in your local development environment"